src/components/nav/Nav.tsx

```diff
@@ -49,17 +48,52 @@ const Ul = styled.ul`
   display: flex;
   flex-flow: row nowrap;
   li {
-    padding: ${(props) =>
+    padding: ${props =>
       props.theme.screens.mmd
         ? "1rem 0.3rem"
         : props.theme.screens.xl
         ? "0.7rem 0.5rem"
         : "1rem 0.65rem"};
-    font-size: ${(props) =>
+    font-size: ${props =>
       props.theme.screens.mmd ? "0.9rem" : props.theme.fontSize.button};
   }
 `;
 
+const SlideDownMenuDesktop = styled.div`
+  position: absolute;
+  display: none;
+  right: 0;
+  background: #ffffff;
+  box-shadow: 0px 4.8px 14.4px rgba(0, 0, 0, 0.18),
+    0px 25.6px 57.6px rgba(0, 0, 0, 0.22);
+  border-radius: 10px;
+  padding: 1rem 0;
+  width: 200px;
+  border-radius: 6px;
+`;
+
+const DropDownNavElement = styled.li`
+  position: relative;
+  &:hover ${SlideDownMenuDesktop} {
+    display: block;
+  }
+`;
+
+const DropDownLinksDesktop = styled(Link)`
+  margin: 0;
+  padding: 0.7rem 1.5rem;
+  display: block;
+  width: 100%;
+  text-align: left;
+  color: #000;
```

Comment:
take colors from theme

Reply:
apply this review for any extra hex or rgb colors...if not present please add to the theme